unyt/tests/test_array_functions.py

```diff
@@ -654,3 +654,27 @@ def test_trim_zeros():
     x1 = [0, 1, 2, 3, 0] * cm
     res = np.trim_zeros(x1)
     assert type(res) is unyt_array
+
+
+def test_any():
+    assert not np.any([0, 0, 0] * cm)
+    assert np.any([1, 0, 0] * cm)
+
+
+def test_append():
+    a = [0, 1, 2, 3] * cm
+    b = np.append(a, [4, 5, 6] * cm)
+    assert type(b) is unyt_array
+    assert 1 * b.units == 1 * cm
```

Comment:
you can compare units directly, no need to multiply by one

Reply:
Oh that's a relief, I don't know why this stuck in my mind. I'll do another PR to simplify my existing tests then